hieradata/common.yaml

```diff
@@ -56,7 +56,7 @@ rhizo::use_ups:                      "no"
 rhizo::use_sip:                      "no"
 rhizo::advice_email:                 "[ 'your@email.address' ]"
 rhizo::sip_central_ip_address:       "127.0.0.1"
-
+rhizo::ntp_servers:                  ['mx.pool.ntp.org', 'pool.ntp.org']
```

Comment:
We don't use either the site_template, or the common, because we are using the private versions. Happy to merge changes to these files, can you make a seperate PR without mixing issues. Thanks!

Reply:
Sure! I'm happy to break the PR down into smaller chunks.